extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/OpenshiftConfig.java

```diff
@@ -41,6 +41,7 @@ public static enum OpenshiftFlavor {
 
     public static enum DeploymentResourceKind {
         Deployment(DEPLOYMENT, DEPLOYMENT_GROUP, DEPLOYMENT_VERSION),
+        @Deprecated(since = "OpenShift 4.14", forRemoval = true)
```

Comment:
The article you link to says

> Also while customers using DeploymentConfigs are encouraged to migrate to Deployments, there are currently (Red Hat OpenShift Container Platform 4.14) no plans to remove the DeploymentConfigs API. Instead the DeploymentConfigs API is being deprecated and therefore no new functionality will be provided and maintenance will be limited to security and critical fixes only.

so `forRemoval = true` seems incorrect?

Reply:
You are probably right. I will fix that.

Reply:
@Ladicek I've fixed this.